docs/csharp/tutorials/console-teleprompter.md

```diff
@@ -376,11 +375,11 @@ private static async Task RunTeleprompter()
 }
 ```
 
-The one new method here is the `Task.WhenAny()` call. That creates a Task
+The one new method here is the @System.Threading.Tasks.Task.WhenAny call. That creates a `Task`
```

Comment:
This doesn't resolve. Should be @System.Threading.Tasks.Task.WhenAny(System.Threading.Tasks.Task[]).